typing/ctype.mli

```diff
@@ -246,6 +246,9 @@ val apply:
 val try_expand_once_opt: Env.t -> type_expr -> type_expr
 val try_expand_safe_opt: Env.t -> type_expr -> type_expr
 
+val path_and_expansion: Env.t -> type_expr -> (Path.t * type_expr) option
```

Comment:
Why have both path and expansion at the same time? I was expecting the implementation to take advantage of doing both operations at once... but it doesn't. Can this be removed in favor of other functions nearby?